src/core/EdFi.DataManagementService.Core/Model/No.cs

```diff
@@ -40,6 +40,8 @@ internal static class No
     /// </summary>
     public static readonly DocumentUuid DocumentUuid = new("00000000-0000-0000-0000-000000000000");
 
+    public static readonly JsonNode EmptyJsonNode = new JsonObject();
```

Comment:
Just call it `JsonNode`. The `Empty` is taken care of when you refer to it in code as `No.JsonNode`